core-active-state.html

```diff
@@ -0,0 +1,101 @@
+<!--
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="../polymer/polymer.html">
+<dom-module id="core-active-state"></dom-module>
+<script>
+  Polymer({
+    is: 'core-active-state',
+
+    properties: {
+      active: {
+        type: Boolean,
+        notify: true,
+        readOnly: true,
+        observer: 'activeChanged',
+        value: false
+      },
+
+      pressed: {
+        type: Boolean,
+        value: function() {
+          this._setPressed(this.pressed || false);
+        },
+        readOnly: true,
+        observer: 'pressedChanged'
+      },
+
+      toggle: {
+        type: Boolean,
+        value: false
+      },
+
+      disabled: {
+        type: Boolean,
+        observer: 'disabledChanged',
+        value: false
+      },
+
+      boundDownHandler: {
```

Comment:
You're right, these could be binding methods instead of anonymous functions (although the outcome is the same). The reason these go in properties is because they would normally be made as default values in e.g., a constructor, but the new way to do that became `configure`, and so the new new way is `properties`.